### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by reefwing**
These six libraries: nexgen_motorshield, nexgen_timer, nexgen_filter, nexgen_AHRS, nexgen_MSP & nexgen_SBUS are being deleted and will be replaced with similar libraries with different branding and repositories.

### CODE REVIEW

**Comment by per1234** [comment]
Hello @reefwing . I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by reefwing** [comment]
Thanks @per1234 - no problem, there is no rush.

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/2231#issuecomment-1362710810

**Review by umbynos — APPROVED** [review]

**Pull request merged**
